src/data/data.js

```diff
@@ -78,6 +78,47 @@ const pageData = {
           },
         ],
       },
+      {
+        component: 'footer',
+        logo: {
+          href: 'http://www.google.com',
+          image: {
+            src: 'footer-logo.png',
+            alt: 'SER HABIL',
+          },
+        },
+        items: [
```

Comment:
I used the word "items" because it appeared in data.js from the beginning, however, I have to wait for the modifications that teachers need to make in data.js. However, it doesn't affect the code, once we will receive the update about the data js, I can change it.